sql/server.go

```diff
@@ -570,7 +570,7 @@ func (s *Server) Set(session *Session, p *parser.Set, args []driver.Datum,
 	// be double quoted.
 	name := strings.ToLower(p.Name.String())
 	switch name {
-	case `"database"`: // Quoted: database is a reserved word
+	case `"database"`, `database`:
```

Comment:
Definitely the wrong place. Keywords are case insensitive. Or they are
supposed to be. I'll track down where we're messing this up in this case.

On Wednesday, July 22, 2015, Tamir Duberstein <notifications@github.com>
wrote:

> In sql/server.go
> <https://github.com/cockroachdb/cockroach/pull/1766#discussion_r35228867>:
>
> > @@ -570,7 +570,7 @@ func (s *Server) Set(session *Session, p *parser.Set, args []driver.Datum,
> >  	// be double quoted.
> >  	name := strings.ToLower(p.Name.String())
> >  	switch name {
> > -	case `"database"`: // Quoted: database is a reserved word
> > +	case `"database"`, `database`:
>
> I don't think this is the right place to do this. @petermattis
> <https://github.com/petermattis> care to weigh in?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/cockroachdb/cockroach/pull/1766/files#r35228867>.
>